### DISCUSSION TITLE
Wyze V3 killed all my camera connections?  How is this possible?

### DISCUSSION THREAD

**Discussion by rocket59** [gtxaspec/wz_mini_hacks#419] [Q&A]
Not entirely sure how to phrase this question, but here goes.

Wyze cam 3, mini hacks installed over 139 firmware with only Hi res video only enabled at default settings and self hosted mode turned on. This was working fine for a while but the stream has been playing up as of a couple of days ago (freezing and lagging).

Re-flashed the hack (just to be sure) and reconnected today. After a few hours all my other cameras (this and 3 V2s with RTSP firmware) had dropped out which required a reboot of the access point (unifi ac pro). Looking at the stats of my AP it seems that the data rate coming off the camera  was like 1.4Mbps which is like twice what the V2 RTSP stream is.

Any ideas as to what's going on? Bandwidth overload problem?

**Comment by dudududodododedede** [comment]
It's a common issue when you many more than 2 or 3 wyzecams
https://www.reddit.com/r/Ubiquiti/comments/jcmo1j/multiple_wyzecams_on_unifi_ap_network_instability/

There's lots of theories on what causes it. Can be very hard to troubleshoot, sometimes the issue occurs immediately when you add the extra device, sometimes everything works fine for days and then you start getting random packet loss (often starts at 1-2% but then gradually climbs to 90%+). The most common cause seems to be related to the router and how it interacts with the WiFi AP

Are you using your ISP's router?

**Reply by rocket59** [reply]
No - its also a ubiquiti. I'll revert to the RTSP beta firmware  and see how that goes. The other 3 cams have been flawless to date.

Edit: More searching shows how widespread an issue this is with Wyze/ubiquiti. Sadly neither party seems keen to get to the bottom of this and no solution seems to be in sight.
